### ISSUE TITLE
Instructions

### ISSUE THREAD

**Issue by njsfield** [RhodesPeter/request-module-workshop#1]
Nice work sir :) 

The readme is very well structured and the instructions are well laid out.

If may be worth clarifying in the readme to run `npm install` to install the request module as a dependency, and clarifying that the the challenge will be to create a module with *similar* features to the request, but with a simpler set of features (outlined in app.js). In *app.js* instead of commenting out the code, perhaps just prompting the cohort to create a 'myRequest' alternate that uses the same parameters to test with...

```javascript
var request = require('request');

function myRequest(){
  /* create your own request module here. 
  It should take a url to make a http GET request, and a callback function with three arguments;
    1. error (String: if an error occurred), 
    2. response(Object; includes the response & statusCode of the request), 
    3. body (String; includes the body of the request)
*/
}

// Helper
function testRequest(module) {
  module('https://jsonplaceholder.typicode.com/users/1', function (error, response, body) {
    console.log('error:', error);
    console.log('statusCode:', response && response.statusCode);
    console.log('body:', body);
  });
}

// request module test
testRequest(request)

// myRequest module test
testRequest(myRequest)
```

**Comment by msachi** [comment]
I would add a (more explicit) reference to Node's core `http.request` (https://nodejs.org/api/http.html#http_http_request_options_callback). That's essentially what the Request module is doing under the hood, so if they want to implement it themselves they'll need to be familiar with it. (Of course you could omit it to make it more challenging, but from my experience it was challenging enough even with the reference to `http.request`. Plus they only have 50 min.)

**Comment by RhodesPeter** [comment]
Fixed with commit 8033ddc581766790e6bc97fc17f2977020aeee68 :)
